lib/custom-logger.js

```diff
@@ -21,15 +21,53 @@
 // we don't want to see emails, usernames and passwords even if crypted and/or hashed
 function defaultRedactionRules() {
   return {
-    paths: ['email', 'password', 'username', '[*].email', '[*].password', '[*].username'],
+    paths: [
```

Comment:
Let's go back to previous implementation since we're not supporting debugInfo anymore